app/clients/sms/twilio.py

```diff
@@ -24,6 +24,15 @@ def get_twilio_responses(status):
 class TwilioSMSClient(SmsClient):
     def __init__(self, account_sid=None, auth_token=None, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        # Twilio docs at.
+        # https://www.twilio.com/docs/usage/webhooks/webhooks-connection-overrides
+        self.callback_url = None
+        self._callback_connection_timeout = 3000  # miliseconds, 10 seconds
+        self._callback_read_timeout = 1500  # miliseconds, 15 seconds
```

Comment:
sorry, wish I'd caught this the last time. can you change these to `milliseconds` (you're missing an l)